### PR TITLE
US1252872 - Add new V1 packages endpoint to PubIQ

### PR DESCRIPTION
**Pull request by chanj14ebsco**
Introduces the new versioned packages endpoint to PubIQ

[US1252872](https://rally1.rallydev.com/#/?detail=/userstory/766601793523&fdp=true): F78567 [I2] PublicationIQ: Add New Query Params, Response Body and Documentation to Packages Endpoint
[US1269512](https://rally1.rallydev.com/#/?detail=/userstory/790727558149&fdp=true): F78567 Adjusted Package URL Logic - PFAPI, PubIQ, gateway, PFUI
[US1252905](https://rally1.rallydev.com/#/?detail=/userstory/766602531895&fdp=true): F78198 [I5] PubIQ: Update documentation for custom package display name
[US1306200](https://rally1.rallydev.com/#/?detail=/userstory/807240664819&fdp=true): PubIQ and rma-rmapigateway: Changes for Freely Available
Commit [f280218](https://github.com/pdestefano/developer_portal_openapi/pull/12/commits/f2802182e1c97a02d60bb35cdea92b33dec3a1d7) is intended as a replacement for PR 21: https://github.com/pdestefano/developer_portal_openapi/pull/21
[US1310092](https://rally1.rallydev.com/#/?detail=/userstory/809768739851&fdp=true): CO: PFAPI - Update default orderby for blank searches

### CODE REVIEW

File: publicationiq.json [hunk 1]
```diff
@@ -5001,6 +5001,145 @@
...
+        },
+        {
+          "name": "orderby",
+          "in": "query",
+          "description": "Valid values are packagename and relevance.",
+          "required": true,
+          "type": "string",
+          "enum": [
+            "relevance",
+            "titlename"
```
Comment (nathancerice) [inline publicationiq.json:5035]: packagename
```suggestion
            "packagename"

```

File: publicationiq.json [hunk 2]
```diff
@@ -5921,6 +6075,147 @@
...
+        },
+        "vendorId": {
+          "type": "integer",
+          "format": "int64"
+        },
+        "vendorName": {
+          "type": "string"
+        },
+        "visibility": {
+          "$ref": "#/definitions/Visibility"
```
Comment (nathancerice) [inline publicationiq.json:6199]: I dont think this is actually being returned

File: publicationiq.json [hunk 3]
```diff
@@ -5921,7 +6076,131 @@
...
+            "type": "string"
+          }
+        },
+        "customDescription": {
+          "type": "string"
+        },
+        "customDisplayName": {
+          "type": "string"
+        },
+        "customerId": {
```
Comment (nathancerice) [inline publicationiq.json:6101]: I dont see customerId coming back on the response either

File: publicationiq.json [hunk 4]
```diff
@@ -5653,6 +5793,21 @@
...
+    "GetPackagesResultV2": {
+      "type": "object",
+      "properties": {
+        "packagesList": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PackageSearchDetails"
+          }
+        },
+        "totalResults": {
```
Comment (nathancerice) [inline publicationiq.json:5805]: also missing the totalSearchTime property on the response

**Review by nathancerice — APPROVED** [review]

**Review by nathancerice — APPROVED** [review]

**Review by nathancerice — APPROVED** [review]

**Review by nwjohnson — APPROVED** [review]

**Review by nathancerice — APPROVED** [review]

**Pull request merged**